src/styles.scss

```diff
@@ -249,3 +249,16 @@ html {
         color: #00ffff;
     }
 }
+.pagination {
+    display: inline-block;
+}
+
+.pagination a {
+    color: black;
+    float: left;
+    padding: 8px 16px;
+    text-decoration: none;
+    transition: background-color 0.3s;
+    border: 1px solid #ddd;
+    cursor: pointer;
+}
```

Comment:
Note that the bottom of the `scss` file are helper classes.
It would be cleaner to separate them from the actual styles.
So this should be above, inside of `.iu`.

Also, unlike `css`, in `scss` you can write this as ->

```scss
.pagination {
    display: inline-block;

    a {
       color: black;
       float: left;
       padding: 8px 16px;
       text-decoration: none;
       transition: background-color 0.3s;
       border: 1px solid #ddd;
       cursor: pointer;
   }
}
```